### PR TITLE
CCM-11696-budget group

### PR DESCRIPTION
**Pull request by charanya-j**
Added a new markdown file and images for budget group documentation.

Ignore other changes. Corrected some inconsistencies in heading capitalization.

### CODE REVIEW

File: docs/cloud-cost-management/5-use-ccm-cost-governance/ccm-budgets/create-budget-group.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+Perform the following steps to create a budget group:
+
+1. In your Harness application, go to **Cloud Costs**, and select **Budgets**.
+2. Click **Create a new Budget Group**.
+3. Enter a budget group name.
+4. Select the budgets to create a group. You could also select multiple budget groups. However, you cannot combine a budget and a budget group to create another group.
+
+  
+:::note
+Note that when selecting budgets, you must ensure that they have the same start date, budget amount type, and period. If you are creating a group of budget groups, the cascading type must also be similar. After selecting the first budget or budget group, the remaining budgets or budget groups that have a different start date, budget amount type, period, or cascading type are disabled.
```
Comment (SudheendraKatte) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/ccm-budgets/create-budget-group.md:33]:
```suggestion
When selecting budgets, you must ensure that they have the same start date, budget amount type, and period. If you are creating a group of budget groups, the cascading type must also be similar. After selecting the first budget or budget group, the remaining budgets or budget groups that have a different start date, budget amount type, period, or cascading type are disabled.
```

File: docs/cloud-cost-management/5-use-ccm-cost-governance/ccm-budgets/create-budget-group.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+:::
+
+3. You could exclude existing budgets or budget groups and include new ones with the same parameters.
+4. If you have selected **Cascading**, depending on the cascading type the budget group amount is redistributed: 
+
+  - **Equally**: The new budget or budget group will be summed up and redistributed across all the budgets in the group.
+  - **Proportionally**: The new budget will be assigned 0% by default. You must edit this value to redistribute. The budgets in the budget group will be recomputed and set to the new budget amounts based on the new ratios.
+  
+:::note
+  Whenever there is a change in the budget values within a nested budget group, the adjustment will cascade upwards, causing a readjustment of the budget ratios all the way to the top. Consequently, the budget amounts in that pathway will increase by a consistent delta amount.
```
Comment (SudheendraKatte) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/ccm-budgets/create-budget-group.md:78]:
```suggestion
Whenever there is a change in the budget values within a nested budget group, the adjustment will cascade upwards, causing a readjustment of the budget ratios all the way to the top. Consequently, the budget amounts in that pathway will increase by a consistent delta amount.
```

**Review by SudheendraKatte — APPROVED** [review]
Approved with minor suggestions.

File: docs/cloud-cost-management/5-use-ccm-cost-governance/ccm-budgets/create-budget-group.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+- Aug, Sep, Oct, Nov - $120
+- Dec - $110
+
+The overall budget is $1200. When this amount is increased to $1800, the difference in the amount, that is $600 is redistributed across months in the same ratio as their initial splits. That is, Jan's budget amount will increase by $40 (80/1200 * 600 = 40).
+
+6. Save the budget group.
+
+## Delete a budget group
+
+1. In **All Budgets**, select the budget group that you want to edit.
```
[the remark this answers is not in the document]
Comment (charanya-j) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/ccm-budgets/create-budget-group.md]: Yes :)

**Pull request merged**